gateway/ln-gateway/src/rpc/mod.rs

```diff
@@ -165,7 +165,11 @@ macro_rules! impl_gateway_request_trait {
 }
 
 impl_gateway_request_trait!(InfoPayload, GatewayInfo, GatewayRequest::Info);
-impl_gateway_request_trait!(ConnectFedPayload, (), GatewayRequest::ConnectFederation);
+impl_gateway_request_trait!(
+    ConnectFedPayload,
+    FederationInfo,
```

Comment:
I think it could be `GatewayInfo` instead, but I'm not sure. @okjodom might know it for sure